src/hotspot/os/aix/os_aix.cpp

```diff
@@ -1126,7 +1128,8 @@ void *os::dll_load(const char *filename, char *ebuf, int ebuflen) {
     dflags |= RTLD_MEMBER;
   }
 
-  void * result= ::dlopen(filename, dflags);
+  const char* error_report = nullptr;
+  void * result = Aix_dlopen(filename, dflags, &error_report);
```

Comment:
Diff to upstream version should get minimized: `void* result;` is in a separate line here: https://github.com/openjdk/jdk/commit/b8ae4a8c0985d1763ac48ba78943d8b992d7be77